src/Umbraco.Core/Services/LocalizedTextServiceExtensions.cs

```diff
@@ -61,5 +61,34 @@ public static string Localize(this ILocalizedTextService manager, string key, Cu
             return variables.Select((s, i) => new { index = i.ToString(CultureInfo.InvariantCulture), value = s })
                 .ToDictionary(keyvals => keyvals.index, keyvals => keyvals.value);
         }
+
+        private static ICultureDictionary _cultureDictionary;
+
+        internal static string UmbracoDictionaryTranslate(this ILocalizedTextService manager, string text)
```

Comment:
On second though, no i'm not going to change it - means some more work. I'll just add TODO notes.